src/platform/3ds/main.c

```diff
@@ -46,23 +46,29 @@ static struct GBAAVStream stream;
 static int16_t* audioLeft = 0;
 static int16_t* audioRight = 0;
 static size_t audioPos = 0;
-static sf2d_texture* tex;
+static struct ctrTexture gbaOutputTexture;
 
 extern bool allocateRomBuffer(void);
 
 static void _postAudioBuffer(struct GBAAVStream* stream, struct GBAAudio* audio);
 
 static void _drawStart(void) {
+	ctrGpuBeginFrame();
 	if (screenMode < SM_PA_TOP) {
-		sf2d_start_frame(GFX_BOTTOM, GFX_LEFT);
+		ctrSetViewportSize(320, 240);
 	} else {
-		sf2d_start_frame(GFX_TOP, GFX_LEFT);
+		ctrSetViewportSize(400, 240);
```

Comment:
Yeah, not as far as I can tell. (It does give you the LCD fb dimensions when you do gfxGetFramebuffer, but there's no reason to call that here.)